### ISSUE TITLE
About the learning rate in finetuning stage

### ISSUE THREAD

**Issue by Andrew-Zhu** [facebookresearch/deit#64]
Hi, I want to finetune the deit_base_patch16_384 model in Imagenet with batch size =64 and 128. 
Basically, I want to follow 

`python run_with_submitit.py --model deit_base_patch16_384 --batch-size 32 --finetune https://dl.fbaipublicfiles.com/deit/deit_base_patch16_224-b5f2ef4d.pth --input-size 384 --use_volta32 --nodes 2 --lr 5e-6 --weight-decay 1e-8 --epochs 30 --min-lr 5e-6`

But I only have one gpu with only 64 or 128 can be set as the batch size. So I use

`python main.py --model deit_base_patch16_224 --batch-size 64  --finetune deit_base_patch16_224-b5f2ef4d.pth --input-size 224 --lr 5e-8 --weight-decay 1e-8 --epochs 30 --min-lr 5e-8 --data-path data/imagenet/ `

--batch-size 64 --min-lr 5e-8
--batch-size 128 --min-lr 5e-7
Am I right?  How can I set the lr respectively? 
Thanks.

**Comment by TouvronHugo** [comment]
Hi @Andrew-Zhu ,
Very good question,
We scale the learning rate linearly according to the batch size with the formula lr_scaled = 5e-4 x Batch size / 512  but I'don't know if it's a very good choice for transformers.
Best,
Hugo

**Comment by Zhu-haow** [comment]
@TouvronHugo Hi, I am doubt about this lr.
python run_with_submitit.py --model deit_base_patch16_384 --batch-size 32 --finetune https://dl.fbaipublicfiles.com/deit/deit_base_patch16_224-b5f2ef4d.pth --input-size 384 --use_volta32 --nodes 2 --lr 5e-6 --weight-decay 1e-8 --epochs 30 --min-lr 5e-6

You use 
linear_scaled_lr = args.lr * args.batch_size * utils.get_world_size() / 512.0

if you set lr = 5e-6  and min-lr = 5e-6,  the lr will less than min-lr.

Besides, this setting is different as you mentioned here.
https://github.com/facebookresearch/deit/issues/45#issuecomment-765213622

**Comment by TouvronHugo** [comment]
Hi @Zhu-haow ,
Thanks for your question,
For fine-tuning you can use SGD or AdamW with different hparams configuration.
For transfer learning experiments I use the configuration of the comment you mention.
Best,
Hugo
